latexify/constants.py

```diff
@@ -14,6 +14,15 @@
 """Latexify module-level constants"""
 
 import ast
+from typing import NamedTuple
+
+
+class Actions(NamedTuple):
```

Comment:
I decided to use `NamedTuple` here because of two benefits.
-   It allows dot access - e.g. `constants.actions.SET_BOUNDS` (vs. `constants.actions[key]` with a `dict` implementation)
-   Tuples are immutable.
-   `NamedTuple` can be written more simpler than a class.

I could have prepared another file `actions.py` to achieve the same interface, but `actions` are essentially constants. Thus I preferred to keep it in `constants.py`.

If you think otherwise, please let me know.